pkg/apis/eksctl.io/v1alpha4/zz_generated.deepcopy.go

```diff
@@ -299,6 +299,13 @@ func (in *NodeGroup) DeepCopyInto(out *NodeGroup) {
 			(*out)[key] = val
 		}
 	}
+	if in.Taints != nil {
```

Comment:
I wonder why is this one getting added... would you mind trying to delete and regenerate it? Seems like it could be a rebase artefact.